### PR TITLE
fixes issues with indexing and rag

### PR DESCRIPTION
**Pull request by thoraxe**
## Type of change

- [ ] Refactor
- [ ] New feature
- [X] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description

* indexer was not using dotenv
* fixes issue in default model for summarizer env
* adds summarizer model example to default env
* prints the embed model being used

## Related Tickets & Documents

N/A

## Checklist before requesting a review

- [X] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
local tests

### CODE REVIEW

File: tools/indexer.py [hunk 1]
```diff
@@ -21,6 +29,8 @@
 else:
   service_context = get_watsonx_context(model="ibm/granite-13b-chat-v1")
```
Comment (bparees) [inline tools/indexer.py]: shouldn't DEFAULT_MODEL be getting referenced somewhere?

Comment (thoraxe) [reply tools/indexer.py]: yes, good catch.

**Comment by thoraxe** [comment]
@bparees made a change in response to your comment

**Comment by bparees** [comment]
lgtm

**Review by bparees — APPROVED** [review]

**Pull request merged**